packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -186,6 +172,16 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
         this.disabledActions.add(actionId);
     }
 
+    private isShown(): boolean {
+        return this.menuElement !== undefined;
```

Comment:
nit: 
```suggestion
        return this.menuElement != null;
```
nit: I think we should avoid using the `undefined` keyword as a value.